tools/text_processing/split_file_to_collection/test-data/mol_2.sdf

```diff
@@ -0,0 +1 @@
+$$$$
```

Comment:
this looks wrong ...

@bernt-matthias we just merged today a fix from @simonbray ... could you please have a look at latest master

Reply:
Sure..

Reply:
hmm... Just thinking about to drop the new `split after` parameter again. In my commits this is determined automagically from the first line.

Reply:
Can you open a PR against the latest master and then we can discuss ...? Thanks Matthias!